### PR TITLE
fix: should ignore empty charaters around comma

### PR DESCRIPTION
**Pull request by DevXiaolan**
see： test cases  "empty_around_comma"  and "empty_side_by_comma"

### CODE REVIEW

File: test/cronstrue.ts [hunk 1]
```diff
@@ -517,6 +517,14 @@ describe("Cronstrue", function () {
   });
 
   describe("errors", function () {
+    it("empty_around_comma", function () {
```
Comment (bradymholt) [inline test/cronstrue.ts:520]: Could you please move these tests to [test/cronParser.ts](https://github.com/bradymholt/cRonstrue/blob/master/test/cronParser.ts#L33-L36)?

File: test/cronstrue.ts [hunk 2]
```diff
@@ -517,6 +517,14 @@ describe("Cronstrue", function () {
...
+      assert.equal(construe.toString('*/5 * * * * ,'), 'Every 5 seconds');
+    });
+
+    it("empty_side_by_comma", function () {
```
Comment (bradymholt) [inline test/cronstrue.ts:524]:
```suggestion
    it("dayOfWeek dangling comma", function () {
```

File: test/cronstrue.ts (hunk 1 above)
Comment (bradymholt) [inline test/cronstrue.ts:520]:
```suggestion
    it("dayOfWeek specified as comma", function () {
```

**Review by bradymholt — CHANGES_REQUESTED** [review]
Thanks for the PR.  I have a few requested changed.

File: test/cronstrue.ts (hunk 1 above)
Comment (DevXiaolan) [reply test/cronstrue.ts]: LGTM

**Comment by DevXiaolan** [comment]
It's that OK, first time solving requested changed online

**Pull request merged**